packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -139,7 +142,7 @@ function getActivationMode(): string {
 }
 
 function registerCommands(
-  extensionContext: vscode.ExtensionContext
+  _extensionContext: vscode.ExtensionContext
```

Comment:
I assume this was done to not hide `extensionContext`?  If so, how about just `context`?  It's a convention we're using elsewhere for function parameters (e.g. `activate()` above).

Reply:
The parameter is never used; it's my understanding that the `_` denotes that and therefor fixes the warning.  But my typescript/javascript isn't strong so let me know if I am mistaken.  I can change it to `_context` for consistency.